### PR TITLE
Felt page

### PR DESCRIPTION
**Pull request by paigewilliams**
[asana task](https://app.asana.com/0/1200144661889303/1208799351016420/f)
Kapture 2024-11-21 at 09 49 56

Not the cutest, but it works!

Some features I would like to add:
- display clicked feature in the sidebar
- custom hover panel
- legend interacting with hovered element
- display a felt map that has a layer for each modality. In the sidebar, select the visible layer via a dropdown

### CODE REVIEW

File: components/layers/layers.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import {
```
Comment (paigewilliams) [inline components/layers/layers.tsx:2]: A lot of this is copied from [here](https://github.com/felt/js-sdk-starter-react/blob/main/src/components/LayersList.tsx) and adapted to our tech stack

File: components/layers/layers.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+} from '@mantine/core';
+import { Layer, LayerGroup } from '@feltmaps/js-sdk';
+import useSWR from 'swr';
+import { IconEye, IconEyeOff } from '@tabler/icons-react';
+import { assembleLayerTree, useFelt, useLiveLayer } from '../../utils/felt';
+
+const Layers: React.FC = () => {
+  const felt = useFelt();
+
+  const fetchLayersAndGroups = async () => {
```
Comment (paigewilliams) [inline components/layers/layers.tsx]: this should probably be wrapped in `useCallback`?

File: components/layers/layers.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    const [layers, layerGroups] = await Promise.all([
+      felt.getLayers().then(layers => layers.filter(Boolean) as Layer[]),
```
Comment (pricealexandra) [inline components/layers/layers.tsx:21]: never seen this filter(Boolean) typescript thing before. cool!

Comment (paigewilliams) [reply components/layers/layers.tsx:21]: yeah me neither! Felt continues to deliver 😎

**Review by pricealexandra — APPROVED** [review]

**Comment by pricealexandra** [comment]
super cool!!!

**Pull request merged**